lib/Controllers/update.js

```diff
@@ -47,10 +28,45 @@ Update.prototype.write = function(req, res, context) {
       context.attributes[a] = req.params[a];
   });
 
+  // Update associated data
+  var self = this,
+      associationPrimKeys = [];
+  if (this.include && this.include.length) {
+    this.include.forEach(function(i) {
+      var instanceModel = i.model || i,
```

Comment:
Now that the MR has been accepted, may be I can rebase this branch and do not check if the model is present in include.
> var instanceModel = i.model || i